docs/developers-landing.md

```diff
@@ -22,4 +13,13 @@ If you want to read top-viewed docs for developers, check out the following conc
 - [Pulsar APIs](pulsar-api-overview.md)
     - [Pulsar admin APIs](admin-api-overview.md)
     - [Pulsar REST APIs](reference-rest-api-overview.md)
-- [Pulsar contribution guide](../../contribute)
\ No newline at end of file
+- [Pulsar contribution guide](../../contribute)
+
+## Advanced topics
+
+If you want to learn how to develop Pulsar related components, check out the following guides.
+
+- [Develop simulation tools](develop-tools.md)
+- [Develop binary protocol](developing-binary-protocol.md)
+- [Develop load manager](develop-load-manager.md)

```

Comment:
This 'develop load manager' page mixes different goals:
- 'Modular load manager': what it is for and how to enable it
- How the modular load manager is implemented
In my opinion, this page is not really about how to develop something for Pulsar.
This is why I don't think this page should be referenced from the 'Developers' page.

Reply:
@heesung-sn @Demogorgon314 
Can I name it to `Develop broker load balancer` (since we plan to [add that topic in `Development`](https://xmind.app/m/ejFj5n))?

<img width="1540" alt="image" src="https://github.com/apache/pulsar-site/assets/50226895/c32addb1-6235-4cc8-9277-8fb599f260e7">

Reply:
yes lgtm.

Reply:
LGTM.